setup.py

```diff
@@ -51,6 +51,7 @@
     "neural-compressor": "optimum-intel[neural-compressor]",
     "graphcore": "optimum-graphcore",
     "habana": "optimum-habana",
+    "diffusers": "diffusers",
```

Comment:
why create a new subpackage? For people to be able to opt-out? Otherwise I would have put it in `exporters`